client/blocks/product-plan-overlap-notices/README.md

```diff
@@ -0,0 +1,45 @@
+Product Plan Overlap Notices
+=======
+
+Product Plan Overlap Notices is a React component for rendering a block with notices.
+
+Those notices will appear when there is a feature overlap between the current plan and the current product, when they are within the provided list of products and plans.
+
+## Usage
+
+```jsx
+import React from 'react';
+import ProductPlanOverlapNotices from 'blocks/product-plan-overlap-notices';
+
+const jetpackPlans = [
+	'jetpack_personal',
+	'jetpack_personal_monthly',
+	'jetpack_premium',
+	'jetpack_premium_monthly',
+	'jetpack_business',
+	'jetpack_business_monthly',
+];
+
+const jetpackProducts = [
+	'jetpack_backup_daily',
+	'jetpack_backup_daily_monthly',
+	'jetpack_backup_realtime',
+	'jetpack_backup_realtime_monthly',
+];
+
+export default class extends React.Component {
+	render() {
+		return (
+			<ProductSelector plans={ jetpackPlans } products={ jetpackProducts } />
```

Comment:
Should this be `ProductPlanOverlapNotices` instead of `ProductSelector`?

Reply:
🤦‍♂ Totally. Fixed with https://github.com/Automattic/wp-calypso/pull/37513/commits/eba7c71f59ec4b079395e3c6c2b3f8fb36ab85ad